lib/packer/builders/amazon.rb

```diff
@@ -12,7 +12,7 @@ def initialize
           'instance_type',
           'region',
           'source_ami',
-          'ssh_username'
+          ['ssh_username', 'communicator']
```

Comment:
Great idea to use LoweredExpectations here! There was already a `min_packer_version` flag on the main `Packer::Config` object but it wasn't really used to good effect. I'm making some changes now, wrapping up the release, I'll include this idea so it forces version 0.8.5 or higher from here on in.